packages/runtime/container-runtime/src/summarizer.ts

```diff
@@ -160,7 +160,7 @@ export class Summarizer extends EventEmitter implements ISummarizer {
         // This will result in "summarizerClientDisconnected" stop reason recorded in telemetry,
         // unless stop() was called earlier
         this.dispose();
-        this.runtime.closeFn();
+        (this.runtime.disposeFn ?? this.runtime.closeFn)()
```

Comment:
@reviewers Is there a cleaner way to do this?
(other than `this.runtime.disposeFn !== undefined ? this.runtime.disposeFn() : this.runtime.closeFn()`)

Reply:
Either of those seems fine 🤷‍♂️